nuxeo-core/nuxeo-core-io/src/main/java/org/nuxeo/ecm/core/io/download/DownloadServiceImpl.java

```diff
@@ -105,6 +106,8 @@
 
     private static final String RUN_FUNCTION = "run";
 
+    private static final Pattern FILENAME_SANITIZATION_REGEX = Pattern.compile(";\\w+=.*$");
```

Comment:
The `$` is not needed either.

Reply:
It's needed to assure that we remove only from the end of the Filename

Reply:
No it's not needed, if you remove it your unit tests still pass. `.*` will match everything until the end.

Reply:
Done